### ISSUE TITLE
Directory: Revert & reverse the "date added" sort order

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1432]
https://fonts.google.com/?sort=date sorts in ascending order (earliest releases at the top, from 2010) but for many years it was descending order (latest releases at the top) which is, I think, more useful.

**Comment by fitojb** [comment]
This looks fixed from here (except for a detail: the Mukta Malar font shows up as the oldest family, before Inconsolata, which is not true).

**Comment by davelab6** [comment]
Yes, this was worked on and fixed but that's a bug that I will fix
